### ISSUE TITLE
Can vpngate.net be disabled???

### ISSUE THREAD

**Issue by jsn0327** [rudiratlos/hotspot#14]
I understand the idea behind providing access to a free VPN service, but after reading through the current hotspot script version, it looks like the OpenVPN section is a complete rewrite and now only supports vpngate.net.  I use a custom VPN config to connect back to my home network while I am traveling and do not have a need for a free VPN service.  Is there a way to set a parameter, which will allow for the old OpenVPN method to work with custom config files instead of using the vpngate.net scoring system and their config files?

**Comment by rudiratlos** [comment]
copy your.ovpn and your.pwd file to /etc/ovpn/client/
hotspot modpar self ovpncfg your.ovpn
hotspot modpar self ovpnwpd your.pwd
hotspot modpar self ovpnrefreshbeforestart no
hotspot modpar self ovpnstart yes

if you don't have a your.pwd, set the parameter to an emtpy string

**Comment by jsn0327** [comment]
Thank you!  That worked.  I recommend adding these instructions to the readme.  You misspelled "opnpwd" in the the instructions though.... I noticed that duplicate OpenVPN tunnels were still being built when running restarting the hotspot, which cause constant disconnect/reconnects, so I created a new function that checks for existing OpenVPN processes and kills them before restarting the hotspot to prevent that.  I've entered a pull request for the changes.  I'm also considering modifying the way that the script checks for the ovpn config files and creating a new option for ovpn private versus the vpngate option, and base the path of the openvpn config files on that parameter so that I don't have to manually change the script parameters each time the script is updated.  I will submit a pull request if/when I get around to doing that.  Thanks a lot for the great work!!
